tests/cli/test_allowed_builtin_toolsets.py

```
@@ -0,0 +1,435 @@
+def test_cli_option_format_validation():
+    """Test that CLI option accepts various formats"""
+    test_cases = [
+        "single",
+        "one,two",
+        "one,two,three",
+        "kubernetes/core,prometheus/core",
+        "aws/ec2, grafana/loki, kubernetes/core",  # With spaces
+        "",  # Empty string
+    ]
+
+    for test_case in test_cases:
```

Ruff (0.12.2)

Loop control variable `test_case` not used within loop body

Rename unused `test_case` to `_test_case`

(B007)